src/include/kdump/uifunctions.rb

```diff
@@ -449,8 +451,8 @@ def tostringKDUMP_SAVE_TARGET
         end 
 
         # ssh
-      elsif Ops.get(@KDUMP_SAVE_TARGET, "target") == "ssh"
-        result = "ssh://"
+      elsif %w[ssh sftp].include?(Ops.get(@KDUMP_SAVE_TARGET, "target"))
```

Comment:
Please use the regular array syntax, `["ssh", "sftp"]`, according to our [style guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md#arrays).